pkg/entities/defi.go

```diff
@@ -289,21 +299,31 @@ func (e *Entity) InDiscordWalletBalances(guildID, discordID string) (*response.U
 		return nil, err
 	}
 
-	gTokens, err := e.repo.GuildConfigToken.GetByGuildID(guildID)
-	if err != nil {
-		err = fmt.Errorf("failed to get guild %s tokens - err: %v", guildID, err)
-		return nil, err
-	}
-	if len(gTokens) == 0 {
-		if err = e.InitGuildDefaultTokenConfigs(guildID); err != nil {
+	var tokens []model.Token
+
+	if guildID == "" {
+		gTokens, err := e.GetGlobalDefaultToken()
```

Comment:
why need a separated entity function here?
we already had `repo.Token.GetDefaultTokens()`

Reply:
i made this function for `configs/tokens/guild_id=""`, here i just reuse it

Reply:
should keep logic handler at entity layer, not handler layer